scripts/anonymization/README.md

```diff
@@ -0,0 +1,220 @@
+# Anonymization
+
+<span style="color:orange"> ANONYMIZATION IS A WORK IN PROGRESS AND DOES CURRENTLY NOT ACTUALLY ANONYMIZE THE DATA. THIS FEATURE WILL BE ADDED LATER </span>
+
+The anonymization module allows to apply privacy mechanisms such as differential privacy or column faking to the data.
+The system will pull data from the JDBC connection, anonymize the data and push it back to the DBMS by creating a new table.
+
+## Config
+
+### Default Parameters
+
+The default parameters must be added inside of the `<table>`-tag. (See below)
+
+| Name        | Default | Possible Value                      | Description                                                                                  |
+| ----------- | ------- | ----------------------------------- | -------------------------------------------------------------------------------------------- |
+| name        | -       | Any string                          | The name of the table                                                                        |
+| epsilon     | 1.0     | Any value > 0                       | The privacy budget. Higher values will decrease the privacy guarantee                        |
+| pre_epsilon | 0.5     | Any value between 0 and epsilon     | The privacy budget spent on preprocessing of the data. Only necessary for continuous columns |
+| algorithm   | mst     | One of: `mst,aim,dpctgan,patectgan` | The differential privacy mechanism applied to the data                                       |
+
+```xml
+<anonymization>
+    <table
+        name="tablename"
+        epsilon="1.5"
+        pre_epsilon="0.75"
+        algorithm="patectgan"
+    />
+</anonymization>
+```
+
+### Column Information
+
+The columns of a table can be split into four categories:
+
+1. `drop` - Columns that are omitted from the DP-mechanism
```

Comment:
Clarifying: 
`drop` -> "omitted" or "removed"?

i.e., are those columns skipped when applying differential privacy but still exist in the dataset at the end as they originally appeared,
or are the "removed" from the dataset (i.e, `ALTER TABLE DROP COLUMN ...`)

Reply:
If the former, maybe call it something else like "passthru" to avoid confusion?

Reply:
renamed to "ignore". Description is in the README. Ignored columns are not anonymized but added back in their original form to the DB